.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,46 @@
+- [ ] I have made sunshine tests for all new `pub` methods.
+- [ ] Code comments are in place for public method along with tricky or non-obvious code segments.
+- [ ] I have performed a self-review of my code
+- [ ] I have not used `unwrap`. Or if I have it is only in tests. 
+- [ ] Any unfinished business is documented with a TODO and a reference to an issue describing what needs to be fixed.
+- [ ] I have only used `expect` or `panic!` in situations where it would imply that there is a bug in the code.
+- [ ] My PR is _not_ updating _any_ dependencies (i.e. no changes to `cargo.lock`). Or if it is, then it _only_ contains the dependency updates and any changes needed to fix compilation and tests.
+- [ ] My changes do not affect the architecture of the protocol. Or if they do these steps must be taken:
+    - [ ] A parallel PR or issue has been open in the [tech-spec repo](https://github.com/zama-ai/tech-spec) (add the link here).
+- [ ] My PR does not contain any breaking changes to the configuration and deployment files. Or if it does then these steps must be taken:
+    - [ ] My PR is labeled with `devops`.
+    - [ ] I have pinged the infra team on Slack (in the MPC channel) and I have updated the intermediate release notes to document this change. 
+    - [ ] I have put a devops person on the PR as reviewer.
+- [ ] My PR does not contain breaking changes to the gRPC interface or data serialized into data in the service gRPC interface. In particular there are no changes to the `extraData` fields. Or if it does the following steps have been taken:
+    - [ ] The PR is marked using `!` in accordance with conventional commits. E.g. `chore!: changed decryption format according to Q3 release`. 
+    - [ ] The gateway teams has been notified about this change. 
+- [ ] I have not changes existing `versionized` structs, nor added new `versionized` structs. Or if I have, these steps must be taken:
+    - [ ] The backwards compatibility tests have been updated and/or new tests added. 
+- [ ] My PR does not contain changes to the critical business logic or cryptographic code. Or if it does then these steps must be taken:
+    - [ ] At least two people must be assigned as reviewers (and eventually approve!) the PR.
+- [ ] I have not added new structs or modified struct to contain private or key data. Or if so then these steps must be taken:
+    - [ ] The `zeroize` and `ZeroizeOnDrop` traits have been implemented to clean up private data.
+- [ ] I have not added data to the public storage. Or if I have, then these steps must be taken:
+    - [ ] I have ensured that the data does _not_ need to be trusted. I.e. it can be validated through a signature or the existence of a digest in the private storage. 
+- [ ] I have not added data to the private storage. Or if I have, then these steps must be taken:
+    - [ ] 
```

Comment:
WIP here

Reply:
Ah sorry. Fixed in a8527285e7721670327f74ca622d2863839b96cc